cpp/ql/src/Likely Bugs/Memory Management/AllocaInLoop.ql

```diff
@@ -185,6 +185,16 @@ class LoopWithAlloca extends Stmt {
     not this.conditionReachesWithoutUpdate(var, this.(Loop).getCondition())
   }
 
+  private Expr getExpr(DataFlow::Node node) {
```

Comment:
I think this hack is perfectly fine if it gives us back the results we want 👍. However, if we start seeing this hack required in more places we should probably discuss if we need some new API on `DataFlow::Node`

Reply:
Agreed.